Apps/Examples/Examples/All Examples/NavigationSimulatorExample.swift

```diff
@@ -0,0 +1,211 @@
+import UIKit
+import MapboxMaps
+
+final class NavigationSimulatorExample: UIViewController, ExampleProtocol {
+    private enum ID {
+        static let routeSource = "route-line-source-id"
+        static let routeLineLayer = "route-line-layer-id"
+        static let casingLineLayer = "route-casing-layer-id"
+    }
+
+    private var mapView: MapView!
+    private var navigationSimulator: NavigationSimulator!
+
+    private lazy var routeSource: Source = {
+        var source = GeoJSONSource()
+        source.data = .geometry(Geometry(sampleRouteLine))
+        source.lineMetrics = true
+
+        return source
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        mapView = MapView(frame: view.bounds)
+        configureMap()
+
+        view.addSubview(mapView)
+        mapView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            mapView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            mapView.leftAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leftAnchor),
+            mapView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+            mapView.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor),
```

Comment:
I wonder if we should use `leading/trailing` instead `left/right` nowadays 🤔 
```suggestion
            mapView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
            mapView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
            mapView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
```


Though e.g. `OrnamentsManager` doesn't seem to support right-to-left locals automatically(ornament view are positioned with absolute constraints).

Reply:
I think it makes sense for Arabic, Hebrew, Persian, Urdu and number of other RTL languages